package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.4.0-beta.0",
+  "version": "0.4.1-dev.20230703082439",
```

Comment:
just need to change this and i think we're all good :)

Reply:
> I think we'll be good to merge this one in itself, but pending we fix the other branch first

Cool! Let me wrap up the #18 first and then I'll come back to this one.

Reply:
#18 merged, let me rebase this one, it will take a few minutes, as this is a branch of branch 👀

Reply:
on standby!

Reply:
Ah, I can't rebase as Gabriel has used merged before, so I used merge too. Easy and quick. it's whatever as we'll squash it all afterward.

I've also removed the `-dev` version, running `npm run version_as_main`